openstack_dashboard/api/glance.py

```diff
@@ -80,7 +80,24 @@ def name(self):
 
     @property
     def project_supported(self):
-        return self.get_is_project_supported()
+        # Use the old way (for now) to get the project_supported attribute
+        # if the site is not showing the Chameleon Support column
+        if not settings.SHOW_CHAMELEON_SUPPORT_COLUMN:
+            return str(self.get_is_project_supported())
+
+        raw = getattr(self._apiresource, 'chameleon-supported', None)
```

Comment:
only thing I can think of, do we want to make this string configurable, or just leave it hardcoded?

in my mind, the main reason that making it configurable is that we set it via config file for glance, but currently hardcode it here.